### PR TITLE
Translation update from Crowdin

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
Depends on review of
- [Stubs PR](https://github.com/microbit-foundation/micropython-microbit-stubs/pull/95)
- [Pyright PR](https://github.com/microbit-foundation/pyright/pull/30)

### CODE REVIEW

File: lang/ui.es-es.json [hunk 1]
```diff
@@ -391,7 +391,7 @@
     "description": "Language option text"
   },
   "less-action": {
-    "defaultMessage": "menos",
```
Comment (microbit-matt-hillsdon) [inline lang/ui.es-es.json:394]: Unexpected translation loss here for less-action, more-action, terms-of-use.

Languages to review: es-es, ko, zh-*

Some other languages look like the string was retranslated though not proposing reviewing those.

Comment (microbit-matt-hillsdon) [reply lang/ui.es-es.json:394]: These were all the master strings without a translation where a duplicate had a translation that we'd previously restored in Crowdin after unexpected structural changes to documents earlier in the year. 

I think this loss is expected in the circumstances and hopefully we've taken steps to ensure it won't recur. I've copied the translations from the duplicates to fix.

I'll do another sync to update this PR.

Comment (microbit-matt-hillsdon) [reply lang/ui.es-es.json]: Done, looks OK now. There's some changes in the same strings in the other languages where the translator filled in the missing translations—they seem good.

**Comment by microbit-matt-hillsdon** [comment]
Just needs the UI testing when it rebuilds.

**Comment by microbit-robert** [comment]
UI looks good. Minor comments on the related PRs - if no action is required then this is good to merge.

**Pull request merged**